### DISCUSSION TITLE
Adding new platform Radxa ROCK Pi 4/4C+

### DISCUSSION THREAD

**Discussion by frebour1** [Ysurac/openmptcprouter#2677] [Ideas]
Is somebody else interrested to have OMR on RAXDA ROCK PI platform ( based on Rockchip RK3399-T ARM Cortex A72).
RockPI 4C+ at 88€

### RK 3399 Specification 
- Dual-core Cortex-A72 up to 1.8GHz
- Quad-core Cortex-A53 up to 1.4GHz
- Mali-T860 GPU
- Dual-channel DDR3/DDR3L/LPDDR3/LPDDR4
- 4K UHD H265/H264/VP9
- HDR10/HLG
- H264 encoder
- Dual MIPI CSI and ISP
- Dual USB Type-C and USB 2.0
- HDMI 2.0a with HDCP 2.2
- MIPI/eDP/RGMII/DP/PCIe
- TrustZone/TEE/DRM

**Comment by Ysurac** [comment]
Raxda ROCK Pi 4 support should be available in latest develop branch.
But you will need to compile it yourself: https://github.com/Ysurac/openmptcprouter/wiki/Create-image-for-unsupported-platform
